Source/Common/Config.cpp

```diff
@@ -70,6 +132,8 @@ namespace FEX::Config {
 
       std::string Program = Args[0];
 
+      Program = Args[0] = RecoverGuestProgramFilename(Args[0], ExecFDInterp, ProgramFDFromEnv);
```

Comment:
* Moving `Args[0]` into the function enables zero-copy behavior for the common path
* `Program` does not have a different value than `Args[0]` on function exit (I think), so it can be a reference
```suggestion
      Args[0] = RecoverGuestProgramFilename(std::move(Args[0]), ExecFDInterp, ProgramFDFromEnv);
      std::string& Program = Args[0];
```

Reply:
Ah, you're right. For some reason I was thinking `Argv[0]` was a reference and was getting modified while reading this and used later when FEXLoader's main pulls the arguments.

Only the following wine executable name searching will see the difference in change beyond what we return. Will fix.

Reply:
Resolved.